pyproject.toml

```diff
@@ -49,3 +44,15 @@ setuptools = ">=68.0.0"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+line-length    = 100
+target-version = "py38"
+output-format  = "full"
+
+[tool.ruff.lint]
+select    = ["E", "F", "I", "W"]
```

Comment:
Additional classes to be added in subsequent issues/PRs. This PR just makes the transition to `ruff`